contents/blog/a-chat-with-sid.md

```diff
@@ -0,0 +1,93 @@
+---
+date: 2020-08-03
+title: The early days of GitLab - a chat with Sid Sijbrandij
```

Comment:
I personally like mostly capitalized titles... "The Early Days of GitLab - A Chat With Sid Sijbrandij"